sass/main.scss

```diff
@@ -4,7 +4,7 @@
     @import "flexbox";
     @import "variables";
 
-    &.sink-views {
+    &.juttle-views {
```

Comment:
kind of worried about css name collision with a commonly used name like `juttle-views`. Perhaps something like `views-layout`